examples/legacy/vault/secret-generator.yaml

```
@@ -0,0 +1,15 @@
+  name: legacy-vault-secrets
+sources:
+  - provider: hashicorp-vault
+    literals:
+      # Single secret values from docker-app namespace (with extra 'data' subfolder)
+      - DATABASE_URL=secret/data/data/docker-app/database-url:value
+      - API_KEY=secret/data/data/docker-app/api-key:value
+      - JWT_SECRET=secret/data/data/docker-app/jwt-secret:value
+    kv:
+      # KV secrets with all key-value pairs (with extra 'data' subfolder)
+      - secret/data/data/docker-app/app-config
+      - secret/data/data/docker-app/feature-flags 
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)